docs/sync.js

```diff
@@ -81,41 +116,60 @@ class Sync {
 }
 
 /**
- * Change information passed when receiving sync 'change' events
+ * Change information passed when receiving sync `'change'` events.
+ *
+ * A ChangeEvent object can only be used within the callback which it is
+ * supplied to, and cannot be stored for use later. In particular, this means
+ * that async functions cannot be used within the callback prior to accessing
+ * the change event. The Realms supplied by the change event do not need to be
+ * explicitly closed.
+ *
  * @memberof Realm.Sync
  */
 class ChangeEvent {
     /**
-     * The path of the changed Realm
+     * The virtual path of the changed Realm. This is the portion of the URL of
+     * the synced Realm after the protocol and the host name.
      * @type {string}
      */
     get path() {}
 
     /**
-     * The changed realm
+     * The changed realm, with the changes applied.
      * @type {Realm}
      */
     get realm() {}
 
     /**
-     * The changed Realm at the old state before the changes were applied
+     * The modified Realm prior to any of the changes being applied. This can
+     * be used in combination with the change indices to read the old values of
+     * any objects which were modified.
+     *
      * @type {Realm}
      */
     get oldRealm() {}
 
     /**
-     * The change indexes for all added, removed, and modified objects in the changed Realm.
-     * This object is a hashmap of object types to arrays of indexes for all changed objects:
+     * The change indexes for all added, removed, and modified objects in the
+     * changed Realm. This object is a hashmap of object types to arrays of
+     * indexes for all changed objects:
+     *
+     * Note that deleting an object in Realm can cause other objects in the
```

Comment:
That doesn't appear to be possible with jsdoc. There's no `@endexample` or equivalent.

Reply:
Sadly true, http://usejsdoc.org/tags-example.html